Source/Settings.as

```diff
@@ -183,6 +191,27 @@ float Setting_Keyboard_Spacing = 10.0f;
 [Setting category="Keyboard" name="Inactive alpha" drag min=0 max=1]
 float Setting_Keyboard_InactiveAlpha = 1.0f;
 
+[Setting category="Keyboard" name="Display arrow symbols"]
+bool Setting_Keyboard_ArrowSymbols = true;
+
+[Setting category="Keyboard" name="Display steer percentage" description="Overrides left/right arrows of setting above"]
+bool Setting_Keyboard_SteerPercentage = false;
+
+[Setting category="Keyboard" name="Display steer percentage symbol"]
+bool Setting_Keyboard_SteerPercentageSymbol = true;
+
+[Setting category="Keyboard" name="Font"]
+string Setting_Keyboard_Font = "DroidSans.ttf";
+
+[Setting
+	category="Keyboard"
+	name="Font size"
+	min=2 max=40]
+int Setting_Keyboard_FontSize = 16;
+
+[Setting category="Keyboard" name="Font color" color]
+vec4 Setting_Keyboard_FontColor = vec4(1, 1, 1, 1);
```

Comment:
Use `TextColor` instead (see also comment on change from `Setting_Gamepad_TextColor` to `Setting_Gamepad_FontColor`)